blinko/docker-compose.yml

```
@@ -0,0 +1,65 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: blinko_app_1
+      APP_PORT: 1111
+      PROXY_AUTH_ADD: "false"
+    networks:
+      - blinko-network
+
+  db:
+    image: postgres:17.2@sha256:3267c505060a0052e5aa6e5175a7b41ab6b04da2f8c4540fc6e98a37210aa2d3
+    hostname: blinko-db
+    mem_limit: 1g
+    cpu_shares: 1024
+    security_opt:
+      - no-new-privileges:true
+    healthcheck:
+      test: ["CMD", "pg_isready", "-q", "-d", "blinko", "-U", "blinkouser"]
+      timeout: 45s
+      interval: 10s
+      retries: 10
+    user: "1000:1000"
+    volumes:
+      - ${APP_DATA_DIR}/data/db:/var/lib/postgresql/data:rw
+    environment:
+      POSTGRES_DB: blinko
+      POSTGRES_USER: blinkouser
+      POSTGRES_PASSWORD: blinkopass
+    restart: on-failure
+    networks:
+      - blinko-network
+
+  app:
+    image: blinkospace/blinko:0.38.5@sha256:6dcd44268d9b158d6e37393b6a9c10266e6fea00d50bdf0899c7480ebb4554b7

```

Update to latest blinko version.
```suggestion
    image: blinkospace/blinko:0.38.6@sha256:e3779d43e987ef65429db43bf06e74da6e0f9896e1d11fb3539d2afb5e85a31b

```